modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -153,6 +153,9 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("routes.mustache", "conf", "routes"));
 
         //App/Utils folder
+        if (!this.controllerOnly && this.useInterfaces) {
```

Comment:
`swagger-codegen-ignore` unfortunately only works for overwrites. It will not prevent the file being created. For our scenario, we are outputting the generated code to the `target` folder (similar to what Play does with its routes), so we don't want to bind Play to any specific implementation (since that will be done by the non-autogenerated code).

I understand having an `interfaceOnly` option would generate a project that would not run as-is, since the implementations would be missing, but that is exactly what we need. We can delete the unnecessary files from the SBT plugin's side, but that seems hacky and will make updating the dependency on the codegen lib more difficult.